lib/create-testing-library-rule/detect-testing-library-utils.ts

```diff
@@ -172,9 +176,9 @@ export function detectTestingLibraryUtils<
      * - it's imported from valid Testing Library module (depends on aggressive
      *    reporting)
      */
-    function isTestingLibraryUtil(
+    function isPotentialTestingLibraryFunction(
```

Comment:
Renaming this method to avoid confusion with the actual method to check if a node is a Testing Library util.